src/configurations/cancercomplexity/synapseConfigs/consortiaHomePage.ts

```diff
@@ -0,0 +1,19 @@
+import { ProgramsProps } from 'synapse-react-client/dist/containers/home_page/programs/Programs'
+
+export const consortiaHomePageConfig: ProgramsProps = {
+  entityId:'syn34267412',
```

Comment:
Do we/stakeholders plan to add a version number later?

Reply:
Good question, has not been needed by other portal (AD)